nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc

```diff
@@ -12,6 +12,33 @@ These release notes track the development of the {secondary-scheduler-operator-f
 
 For more information, see xref:../../../nodes/scheduling/secondary_scheduler/index.adoc#nodes-secondary-scheduler-about_nodes-secondary-scheduler-about[About the {secondary-scheduler-operator}].
 
+[id="secondary-scheduler-operator-release-notes-1.1.4"]
+== Release notes for {secondary-scheduler-operator-full} 1.1.4
+
+Issued: 25 November 2024
```

Comment:
@bergerhoffer just one comment, could you  please help change the date as this is changed based on my request yesterday ?, thank you !!

Reply:
Updated, thanks! (Will take a few mins for the preview to update)